packages/server/graphql/intranetSchema/mutations/runScheduledJobs.ts

```diff
@@ -65,19 +84,13 @@ const processMeetingStageTimeLimits = async (job: ScheduledJobMeetingStageTimeLi
       notification
     })
   }
-
-  // get the meeting
-  // get the facilitator
-  // see if the facilitator has turned on slack notifications for the meeting
-  // if so, send the facilitator a slack notification
-  // if not, send the facilitator an in-app notification
 }
 
 const jobProcessors = {
   MEETING_STAGE_TIME_LIMIT_END: processMeetingStageTimeLimits
 }
```

Comment:
if we cast this `as const` then we don't need `job as any` down below. this'll keep our types nice & strong

Reply:
I couldn't get that to work. I needed to recast `job` itself, but do it based on what was in `jobProcessors`. I decided to do this instead:

```typescript
processor(job as Parameters<typeof processor>[0], {dataLoader}).catch(console.log)
```